framework/doc/content/source/meshgenerators/MeshGenerator.md

```diff
@@ -142,3 +142,41 @@ GeneratedMeshGenerator 'gmg':    Bounding box maximum: (x,y,z)=(       1,
 GeneratedMeshGenerator 'gmg':    Bounding box delta: (x,y,z)=(       1,        1,        0)
 GeneratedMeshGenerator 'gmg':   Global mesh volume = 1
 ```
+
+The parameter [!param](/Mesh/GeneratedMeshGenerator/output) output a mesh file immediately after it is generated. It can be used to view the intermediate mesh block during mesh generation.
```

Comment:
```suggestion
The parameter [!param](/Mesh/GeneratedMeshGenerator/output) enables the output of a mesh file immediately after it is generated. It can be used to view the intermediate meshes during mesh generation.
```